### PR TITLE
GUA 748 format activity log

### PR DESCRIPTION
**Pull request by divyaparameswaran**
Adds a Lambda that:

Reads in an from the sqs queue and formats the data into a structure to write in dynamodb
The output is written to a subsequent SQS queue

### CODE REVIEW

File: lambda/format-activity-log/format-activity-log.ts
Comment (DougsCER) [inline lambda/format-activity-log/format-activity-log.ts:1]: Would this be the correct place to truncate the log if there were too many activities in it?

Comment (DougsCER) [reply lambda/format-activity-log/format-activity-log.ts:1]: the alternative is we don't truncate in the DB, but we do in the frontend

Comment (DougsCER) [reply lambda/format-activity-log/format-activity-log.ts:1]: agreed on slack this lambda should not allow more than 100 activities in each record.  Upon receipt of activity 101  "truncated" field should be set to "true".
When further events are received no DB update should be performed.

Comment (divyaparameswaran) [reply lambda/format-activity-log/format-activity-log.ts:1]: Thank you I have added this and test .

File: lambda/format-activity-log/format-activity-log.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  }
+};
+
+export const formatIntoActivitLogEntry = (
+  userActivityLog: UserActivityLog
+): ActivityLogEntry => {
+  if (userActivityLog.activityLogEntry === undefined) {
+    return createNewActivityLogEntryFromTxmaEvent(userActivityLog.txmaEvent);
+  }
+  if (userActivityLog.activityLogEntry.activities.length < 99) {
```
Comment (DougsCER) [inline lambda/format-activity-log/format-activity-log.ts:78]: why less than 99?  should it not be < 100

Comment (DougsCER) [reply lambda/format-activity-log/format-activity-log.ts:78]: to get a maximum of 100 activities in an event

Comment (divyaparameswaran) [reply lambda/format-activity-log/format-activity-log.ts:78]: `Oh yes . Not sure what I was thinking

**Pull request merged**